lib/dialects/snowflake/data-types.ts

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+import moment from 'moment';
+import { kSetDialectNames } from '../../dialect-toolbox';
+import * as BaseTypes from '../abstract/data-types';
+import type {
+  AcceptableTypeOf,
+  StringifyOptions,
+} from '../abstract/data-types';
+
+// TODO [>6]: Why are we changing abstract data-types???
```

Comment:
Do we all agree that this will only ever land in v7 and onwards?

Reply:
Yeah, this PR will not land in v6. But I think the idea of this TODO is that it will be fixed in a next PR but before the release of v7